src/Integration.Vsix/SonarLintDaemonPackage.cs

```diff
@@ -90,6 +88,7 @@ private async System.Threading.Tasks.Task InitAsync()
                 await CFamilyReproducerCommand.InitializeAsync(this, logger);
 
                 cFamilyPreCompiledHeadersEventListener = await this.GetMefServiceAsync<IPreCompiledHeadersEventListener>();
+                activeSolutionChangedCallback = await this.GetMefServiceAsync<IActiveSolutionChangedCallback>();
```

Comment:
Another item in the dumping ground... I remembered why we used this for the CFamily stuff: because the integration package only applies for .net stuff, not cfamily/js.

Side note: the callbacks won't be called until this package loads, so it is possible that a solution is opened/closed at the start of VS before this package loads, and the eslint-bridge will be left behind.

Reply:
Hhmm. The daemon package loads when the shell is initialized, but since it's done in the background there can be quite a delay before VS actually gets round to loading it.

I was hoping to avoid having the analyzers act as observers too. However, I've hit a similar problem with disabling rules - we need to call `init-linter` again when the user settings change i.e. something needs to monitor the user settings file for changes :-(